src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -122,7 +122,7 @@ class TxQ_test : public TestSuite
     }
 
 public:
-    void run()
+    void testQueue()
```

Comment:
Weird. I could swear I tried that in an earlier instance and it puked because of the `const` conversion, but it seems to work fine now.